newIDE/app/src/UI/SortableVirtualizedItemList/index.js

```diff
@@ -86,6 +87,14 @@ export default class SortableVirtualizedItemList<Item> extends React.Component<
     );
   }
 
+  // Workaround for whitespace issue cause by react-visualized

```

Comment:
I'm not sure I understand exactly why this works.  Is there an issue opened on `react-virtualized` GitHub repository that we can link? (and if so, maybe just updating `react-virtualized` would work?